streams/shared/src/main/scala/scalaz/zio/stream/ZStream.scala

```diff
@@ -379,10 +412,43 @@ trait ZStream[-R, +E, +A] extends Serializable { self =>
     override def fold[R2 <: R1, E2 >: E1, B1 >: B, S]: Fold[R2, E2, B1, S] =
       ZManaged.succeedLazy { (s, cont, g) =>
         self.fold[R2, E2, A, S].flatMap(fold => fold(s, cont, (s, a) => f(a).flatMap(g(s, _))))
-
       }
   }
 
+  final def mapMPar[R1 <: R, E1 >: E, B](n: Int)(f: A => ZIO[R1, E1, B]): ZStream[R1, E1, B] =
```

Comment:
Would you mind renaming `n` to `outputBuffer` for consistency with `flatMapPar`? What do you think of using the same default value?

Reply:
Oh but it has a different meaning? This `n` is the same as in `flatMapPar` - the maximum number of concurrent executions of `f`.

Reply:
But maybe it's a good idea to (at some point) go through and change `n` to `parallelism` in all the combinators that support it (including stuff like `foreachParN`, etc.).

Reply:
Yes, that'd be good. Happy to do that in a followup.